src/test/java/studyconnect/member/dto/LoginRequestTest.java

```diff
@@ -0,0 +1,17 @@
+package studyconnect.member.dto;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.studyconnect.member.dto.LoginRequest;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.studyconnect.member.dto.LoginRequest' import. Should be before 'org.junit.jupiter.api.Test'.